src/view.c

```diff
@@ -2142,25 +2142,38 @@ view_snap_to_edge(struct view *view, enum lab_edge edge,
 
 	view_set_shade(view, false);
 
-	if (across_outputs && view->tiled == edge && view->maximized == VIEW_AXIS_NONE) {
-		/* We are already tiled for this edge; try to switch outputs */
-		output = output_get_adjacent(view->output, edge, /* wrap */ false);
+	if (lab_edge_is_cardinal(edge) && view->maximized == VIEW_AXIS_NONE) {
+		enum lab_edge invert_edge = lab_edge_invert(edge);
+		enum lab_edge parallel_mask = edge | invert_edge;
```

Comment:
Suggest describing (or giving examples of) `parallel_mask`, `parallel_tiled` and `orthoganol_tiled` to make it easier for the reader.